### ISSUE TITLE
Fix passing filter_min_reads as int

### ISSUE THREAD

**Issue by prot-devin** [nf-core/fastquorum#62]
### Description of the bug

When analyzing non-duplex reads it makes sense to set filter_min_reads to an int, however. when setting `--filter_min_reads 1` you will encounter the following error:
` --filter_min_reads: expected type: String, found: Integer (1)`

even passing --filter_min_reads "1" does not seem to solve this issue.

### Command used and terminal output

```console
nextflow run nf-core/fastquorum --input $sample_sheet --outdir $outdir --fasta $reference_fasta --mode rd --groupreadsbyumi_strategy Identity --groupreadsbyumi_edits 0 --duplex_seq false --filter_min_reads 1 -profile docker -resume

Error message: 
--filter_min_reads: expected type: String, found: Integer (1)
```

### System information

Nextflow version: 24.04.2
Hardware: HPC
Executor: slurm
Container: docker/singularity
OS: Ubuntu
nf-core/fastquorum v1.0.1dev

**Comment by prot-devin** [comment]
@SPPearce has suggested a workaround of `--filter_min_reads '"1"'` for the time being

**Comment by SPPearce** [comment]
Fixed in #66
